lib/component.js

```diff
@@ -427,19 +427,22 @@ class Component extends WebComponent {
     const attrsSchema = this.constructor.attrsSchema;
     if (attrsSchema.hasOwnProperty(attr)) {
       const attrSchema = attrsSchema[attr];
```

Comment:
nit: I would just normalize it once here, so the rest of this block doesn't have to know about the shortcut. e.g.
```javascript
let attrSchema = attrsSchema[attr];
if (typeof attrSchema === `string`) {
  attrSchema = {type: attrSchema};
}
```